Bamboo/docker/Dockercompose.local.yaml

```diff
@@ -1,4 +1,4 @@
-version: "3.9"
+# version: "3.9"
```

Comment:
Was there an issue in specifying the docker compose version?

Reply:
Yep they said using version was outdated, so I commented it out. Its also possible my version is too new or too old.

Reply:
Oh lol ok we can use the newer version, I stuck to 3.9 cause usually itll default to a lower version -- but that was out of habit